app/code/Magento/Cms/Test/Mftf/ActionGroup/SaveAndDuplicateCMSPageWithSplitButtonActionGroup.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="SaveAndDuplicateCMSPageWithSplitButtonActionGroup">
```

Comment:
Please, prefix this action group with the area, so we have in result `AdminSaveAndDuplicateCMSPageWithSplitButtonActionGroup`. Thank you

Reply:
@rogyar I have resolved as per comments.